Documentation/release-notes/4567.md

```diff
@@ -0,0 +1,18 @@
+### Preliminary JDK 11 Support
+
+Xamarin.Android can now use JDK 11 to build libraries and applications.
+
+To do so:
+
+ 1. Install JDK 11
+ 2. Set the `$(JavaSdkDirectory)` MSBuild property to refer to the root
+    of the JDK installation from (1).
+
+#### Known Issues
+
+When JDK 11 is used, the `@(JavaDocJar)` Build action is not supported,
```

Comment:
It could be handy to create a GitHub issue for this known issue so that it will already have a tracking location, but it's OK to skip that for now too.